libraries/AP_Common/AP_Common.h

```diff
@@ -23,7 +23,10 @@
 #include <AP_HAL/AP_HAL_Boards.h>
 #include <stdint.h>
 #include <stdlib.h>
+
+#if !defined(__ANDROID__) && !defined(__BORLANDC__)
```

Comment:
and add stdbool.h to the missing/  directory. However.... for this particular header: we can remove it from everywhere since we are using C++, not C. The only exceptions are:
  - libraries/AP_Module/AP_Module_Structures.h since it may be used externally
  - libraries/AP_Module/examples/ModuleTest/module/moduletest.c since it's a C source